libraries/botbuilder-lg/src/expander.ts

```diff
@@ -536,6 +535,21 @@ export class Expander extends AbstractParseTreeVisitor<string[]> implements LGTe
             return standardFunction;
         }
 
+        const pointIndex = name.indexOf('.');
+        if (pointIndex > 0) {
+            const alias = name.substr(0, pointIndex);
+            if (alias in this.templates.namedReferences) {
```

Comment:
This could be:

```typescript
const realTemplate = this.templates.namedReferences[alias];
if (realTemplate) {
  const realTemplateName = ...;
  FunctionUtils.apply(this.evaluateWithTemplates(realTemplateName, realTemplate)), ReturnType.Object);
}
```

It avoids the double name lookup.